### PR TITLE
846230 -UG documentation for Volume 3, 2023

### PR DESCRIPTION
**Pull request by jeyalakshmit**
Hi All,

Added the UG Documentation for below Volume 3 features
1.Remove Specific Keys from the Existing Document Information
2.HTML Header and Footer
3.Get images from the existing signed signature field

Regards,
Jeyalakshmi T

### CODE REVIEW

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 1]
```diff
@@ -1732,6 +1732,74 @@ document.Close(True)
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PDF-Examples/tree/master/HTML%20to%20PDF/Blink/HTML-to-PDF-rotate-page).
 
+## HTML Header and Footer
+
+You can set Html string as header and footer when converting HTML to PDF, utilize the **HtmlHeader** and **HtmlFooter** properties in the [BlinkConverterSettings](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.BlinkConverterSettings.html) class. Please refer to the following code example.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:1737]: When converting HTML to PDF, you have the option to set an HTML string as both the header and footer. To do this, make use of the HtmlHeader and HtmlFooter properties within the [BlinkConverterSettings](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.BlinkConverterSettings.html) class.

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 2]
```diff
@@ -1732,6 +1732,74 @@ document.Close(True)
...
+document.Save(fileStream)
+document.Close(True)
+
+{% endhighlight %}
+
+{% endtabs %}
+
+You can download a complete working sample from GitHub.
+
+N> Need to set the html margin top and bottom value based on the html header footer height and margin value.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:1801]: Adjust the HTML margin top and bottom values according to the height of the HTML header and footer, as well as the margin settings.

File: File-Formats/PDF/Working-with-DigitalSignature.md [hunk 3]
```diff
@@ -3982,4 +3982,67 @@ document.Close(True)
 
 {% endtabs %}
 
-You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PDF-Examples/tree/master/Digital%20Signature/Retrieve-revocation-certificate-information-from-digital-signature-embed-timestamp).
\ No newline at end of file
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PDF-Examples/tree/master/Digital%20Signature/Retrieve-revocation-certificate-information-from-digital-signature-embed-timestamp).
+
+## Get images from the existing signed signature field
+
+You can get the images from the existing signed signature field’s appearance using the **GetImages** method in [PdfLoadedSignatureField](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Parsing.PdfLoadedSignatureField.html) class. The following code example explains this.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-DigitalSignature.md:3989]: To obtain images from the appearance of an existing signed signature field, you can utilize the GetImages method within the PdfLoadedSignatureField class. The code example below provides a clear demonstration of this functionality.

File: File-Formats/PDF/Working-with-Document.md [hunk 4]
```diff
@@ -648,6 +648,85 @@ document.Close(True)
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PDF-Examples/tree/master/PDF%20Document/Change_existing_PDF_properties). 
 
+## Remove the specified key from the document information in existing PDF document
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-Document.md:651]: Remove specific key from the existing document information

File: File-Formats/PDF/Working-with-Document.md [hunk 5]
```diff
@@ -648,6 +648,85 @@ document.Close(True)
...
+
+To remove specific details from the existing document information, the **Remove** method is utilized. The following code example explain this.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-Document.md:653]: To remove specific details from the existing document information, you can make use of the Remove method. The following code example provides an explanation of how to do this.

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md (hunk 1 above)
Comment (jeyalakshmit) [reply File-Formats/PDF/Convert-HTML-To-PDF/features.md:1737]: changed

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md (hunk 2 above)
Comment (jeyalakshmit) [reply File-Formats/PDF/Convert-HTML-To-PDF/features.md:1801]: changed

File: File-Formats/PDF/Working-with-DigitalSignature.md (hunk 3 above)
Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-DigitalSignature.md:3989]: changed

File: File-Formats/PDF/Working-with-Document.md (hunk 4 above)
Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-Document.md:651]: changed

File: File-Formats/PDF/Working-with-Document.md (hunk 5 above)
Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-Document.md:653]: changed

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 6]
```diff
@@ -1732,6 +1732,76 @@ document.Close(True)
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PDF-Examples/tree/master/HTML%20to%20PDF/Blink/HTML-to-PDF-rotate-page).
 
+## HTML Header and Footer
+
+When converting HTML to PDF, you have the option to set an HTML string as both the header and footer. To do this, make use of the **HtmlHeader** and **HtmlFooter** properties within the [BlinkConverterSettings](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.BlinkConverterSettings.html) class.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:1737]: You have provided the code for HTML file not HTML string. So change the sentence

File: File-Formats/PDF/Working-with-Document.md [hunk 7]
```diff
@@ -648,6 +648,85 @@ document.Close(True)
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PDF-Examples/tree/master/PDF%20Document/Change_existing_PDF_properties). 
 
+## Remove specific key from the existing document information
+
+To remove specific details from the existing document information, you can make use of the **Remove** method. The following code example provides an explanation of how to do this.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-Document.md:653]: To remove specific details from the existing document information, you can make use of the **Remove** method of DocumentInformation class.

Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-Document.md:653]: Changed

**Comment by jeyalakshmit** [comment]
rebuild

**Comment by kannans24** [comment]
rebuild

**Comment by kannans24** [comment]
rebuild

File: File-Formats/PDF/Working-with-Document.md [hunk 8]
```diff
@@ -648,6 +648,85 @@ document.Close(True)
...
+'Save the document.
+document.Save("Output.pdf")
+'Close the document.
+document.Close(True)
+
+{% endhighlight %}
+
+{% endtabs %}
+
+Download a complete working sample from GitHub.
```
Comment (chinnumuniyappan) [inline File-Formats/PDF/Working-with-Document.md:728]: Add GitHub sample link

File: File-Formats/PDF/Working-with-DigitalSignature.md [hunk 9]
```diff
@@ -3982,4 +3982,66 @@ document.Close(True)
...
+    images(i).Save("Image.jpg", Imaging.ImageFormat.Png)
+Next
+'Close a PDF document.
+ldoc.Close(True)
+
+{% endhighlight %}
+
+{% endtabs %}
+
+Download a complete working sample from GitHub.
```
Comment (chinnumuniyappan) [inline File-Formats/PDF/Working-with-DigitalSignature.md:4045]: Add GitHub sample link

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 10]
```diff
@@ -1732,6 +1732,76 @@ document.Close(True)
...
+Dim fileStream As FileStream = New FileStream("Output.pdf", FileMode.CreateNew, FileAccess.ReadWrite)
+'Save and close a PDF document.
+document.Save(fileStream)
+document.Close(True)
+
+{% endhighlight %}
+
+{% endtabs %}
+
+Download a complete working sample from GitHub.
```
Comment (chinnumuniyappan) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:1801]: Add GitHub sample link

**Review by chinnumuniyappan — CHANGES_REQUESTED** [review]

File: File-Formats/PDF/Working-with-DigitalSignature.md (hunk 9 above)
Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-DigitalSignature.md]: Removed

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md (hunk 10 above)
Comment (jeyalakshmit) [reply File-Formats/PDF/Convert-HTML-To-PDF/features.md]: Removed

File: File-Formats/PDF/Working-with-Document.md (hunk 8 above)
Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-Document.md]: Removed

**Comment by kannans24** [comment]
rebuild

**Review by chinnumuniyappan — APPROVED** [review]

**Pull request merged**